src/test/regress/sql/view_propagation.sql

```diff
@@ -497,6 +497,30 @@ DROP VIEW IF EXISTS axx.temp_view_to_drop;
 DROP MATERIALIZED VIEW IF EXISTS axx.temp_view_to_drop;
 DROP MATERIALIZED VIEW IF EXISTS axx.temp_view_to_drop;
 
+-- Test the GUC citus.local_view_dependency_checks
+SET search_path to view_prop_schema;
+
+CREATE TABLE dist_t (a int);
+SELECT create_distributed_table('dist_t', 'a');
+
+CREATE TABLE local_t (a int);
+
+SET citus.local_view_dependency_checks TO false;
+-- views will be created locally with warnings
+CREATE VIEW vv1 as SELECT * FROM local_t;
+CREATE OR REPLACE VIEW vv2 as SELECT * FROM vv1;
+-- view will fail due to local circular dependency
+CREATE OR REPLACE VIEW vv1 as SELECT * FROM vv2;
+
+SET citus.local_view_dependency_checks TO true;
```

Comment:
can you also add a test with `CREATE VIEW v_local_only AS SELECT 1`

and show that we can create that locally (e.g., does not have any rows in `select * from pg_dist_object where objid = 'v_local_only'::regclass;`)